projects/igniteui-angular/src/lib/core/styles/components/drop-down/_drop-down-theme.scss

```diff
@@ -212,21 +212,21 @@
 
     %igx-drop-down-wrapper {
```

Comment:
Where does this get extended? You have deleted all references of it from the component styles.